### PR TITLE
update registry by adding all known packages

### PR DESCRIPTION
**Pull request by amtoine**
followup to #56 

## Description
this should update the `registry.nuon` file thanks to the content of previous `index.nuon` (see cbca144 for the pipeline used to update the registry).

this PR uses the latest version of each package.

## try it out
```nushell
nupm search --registry ./registry.nuon explore
    | into record
    | update pkgs { into record }
    | table --expand
```
```
────┬──────────────────────────────────────────────────────
name│registry
path│./registry.nuon
    │────────┬─────────────────────────────────────────────
pkgs│name    │nu_plugin_explore
    │version │0.1.2
    │url     │https://github.com/:amtoine/nu_plugin_explore
    │revision│0.1.2
    │path    │.
    │type    │git
    │────────┴─────────────────────────────────────────────
────┴──────────────────────────────────────────────────────
```

### CODE REVIEW

File: registry.nuon [hunk 1]
```diff
@@ -26,21 +26,166 @@
...
+            version,
+            url,
+            revision,
+            path
+        ];
+        [
+            nu_plugin_audio_hook,
+            "0.1.12",
+            "https://github.com/FMotalleb/nu_plugin_audio_hook",
+            "0.1.12",
```
Comment (kubouch) [inline registry.nuon:40]: I think the version and revision are a bit mixed on this one. The last version and revision seems to be "v0.1.2".

File: registry.nuon [hunk 2]
```diff
@@ -26,21 +26,166 @@
...
+            .
```
Comment (kubouch) [inline registry.nuon:41]: I feel like we should allow `null` here.

Comment (amtoine) [reply registry.nuon:41]: yeah, could allow that as well :+1:

File: registry.nuon (hunk 1 above)
Comment (amtoine) [reply registry.nuon:40]: i did not look at these other packages, as i do not maintain them.
i can have a look real quick

**Review by kubouch — APPROVED** [review]

**Pull request merged**